dev-python/tablib/tablib-3.0.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Format-agnostic tabular dataset library"
+HOMEPAGE="https://pypi.org/project/tablib/ https://github.com/jazzband/tablib/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+BDEPEND="
+	test? (
+		dev-python/pytest
+		dev-python/pytest-cov
+		dev-python/odfpy
+		dev-python/openpyxl
+		dev-python/markuppy
+		dev-python/pandas
+		dev-python/pyyaml
+		dev-python/tabulate
+		dev-python/xlrd
+		dev-python/xlwt
+	)
+"
+
+distutils_enable_tests pytest
+
+python_test() {
+	epytest --deselect MarkupPy
```

Comment:
Why do you deselect it if you added it as a dep anyway?

Reply:
Sorry, typo

Reply:
Sorry, typo